extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -12,6 +12,7 @@ import RX = require('reactxp');
 
 export interface VirtualListCellInfo {
     key: string;
+    onScreenChanged?: (isOnScreen: boolean) => void;
```

Comment:
An interface change like this should be accompanied by an update to the documentation (VirtualListView.md).